Makefile

```diff
@@ -26,6 +26,12 @@ YQ_INTERNAL := go run $(MOD_FLAGS) ./vendor/github.com/mikefarah/yq/v2/
 KUBEBUILDER_ASSETS := $(or $(or $(KUBEBUILDER_ASSETS),$(dir $(shell command -v kubebuilder))), /usr/local/kubebuilder/bin)
 export KUBEBUILDER_ASSETS
 
+ifeq ($(shell go env GOARCH), ppc64le)
```

Comment:
Should this just be `GOARCH := $(shell go env GOARCH)`?

Reply:
so the GOARCH for amd64 devices will be amd64 instead of 386, so I had to use the if to make sure other than ppc64le it'd be 386. this was found when the tests were ran. it seems these prow arches are amd64

Reply:
@ecordell please advise on how you would like this to be handled if that's not satisfactory

Reply:
@ecordell bump :)